src/Sulu/Bundle/AdminBundle/Resources/js/components/ImageRectangleSelection/ImageRectangleSelection.js

```diff
@@ -3,7 +3,7 @@ import {action, computed, observable} from 'mobx';
 import log from 'loglevel';
 import {observer} from 'mobx-react';
 import React from 'react';
-import RectangleSelection from '../RectangleSelection';
+import {RectangleSelection} from '../RectangleSelection/RectangleSelection';
```

Comment:
Because the ImageRectangleSelection passes it's own `containerWidth` and `containerHeight` to the CircleSelection and this is only possible, if i use the "real" CircleSelection and not the wrapper `withContainerSize(CircleSelection)`

Reply:
You know that we are talking about the `RectangleSelection` here, and not about the `CircleSelection`?

Reply:
Oh, my fault. But that doesn't change anything, since those two are structured the same way.

Reply:
I still think that this change does not make any sense here, and that it is the reason that you suddenly have to pass `containerWidth` and `containerHeight` in here, which hadn't to be done before, and it was also working. So if that does not work anymore, it is because of some change you've been making in this PR.

Reply:
Yes I agree and the reason for this is, that I had to change the `RectangleSelection` a lot to make it work with the ImageMap. But I don't understand what's the problem here?

Reply:
The "problem" is that the export you are using here was only used in Tests until now, and I would prefer to not even use it there, so that we can remove that export completely, so that there is less public facing API we would have to support. That's the reason I would try not to use that export at all. But maybe that issue here resolves itself when we finish what we've agreed on in [the other comment](https://github.com/sulu/sulu/pull/5434#discussion_r494962531).